src/errorpage.cc

```diff
@@ -712,6 +712,8 @@ ErrorState::ErrorState(HttpRequest * req, HttpReply *errorReply) :
         request = req;
         src_addr = req->client_addr;
     }
+
+    ale = anAle;
```

Comment:
To avoid double initialization, should not we init ale in the parameter list?

Reply:
However we could add ALE parameter to that private ErrorState constructor then...

Reply:
Done in commit b143820.